packages/ui/src/composables/useCSSVariables.ts

```diff
@@ -2,5 +2,8 @@ import { computed } from 'vue'
 import kebab from 'lodash/kebabCase.js'
 
 export const useCSSVariables = (prefix: string, cb: () => Record<string, string>) => {
-  return computed(() => Object.entries(cb()).map(([key, value]) => ({ [`--${prefix}-${kebab(key)}`]: value })))
+  return computed(() => Object.entries(cb()).reduce((acc, [key, value]) => {
+    acc[`--${prefix}-${kebab(key)}`] = value
```

Comment:
Would be nice to have here return value typing as in `useBem`. But up to you.

Reply:
To much effort for it...